### PR TITLE
DEV-461 Add option to phctl to run inline

### PR DESCRIPTION
**Pull request by aelkiss**
Having a global option for thor is a little bit clunky, but using the "class_option" on the parent class and then handling the option in a single place that the subcommands inherit from seems to work. If there is a better way to do this with Thor, I couldn't find it.

### CODE REVIEW

File: lib/phctl.rb [hunk 1]
```diff
@@ -122,16 +138,23 @@ def uncommitted_holdings
...
-      Jobs::Common.perform_async("Reports::OCLCRegistration", options, organization)
+      run_common_job(Reports::OCLCRegistration, options, organization)
     end
   end
 
   class PHCTL < Thor
+    # Run inline instead of with sidekiq
+    class_option :inline, type: :boolean
+
+    def self.exit_on_failure?
```
Comment (aelkiss) [inline lib/phctl.rb:154]: This suppresses a warning from Thor about not having this here.

Comment (aelkiss) [reply lib/phctl.rb:154]: And probably prevents cron jobs etc from falsely thinking they're successful.

**Comment by aelkiss** [comment]
Note that coveralls still posted its comment but did not include the status check (which would have been OK in this case)

**Review by mwarin — APPROVED** [review]
That's really cool.

**Pull request merged**